views/layout.jade

```diff
@@ -1,11 +1,11 @@
-!!! 5
+doctype html
 html
   head
     title #{settings.sitename} - #{title}
     link(rel='stylesheet', href='/stylesheets/bootstrap.min.css')
-    style(type='text/css')
-      body{padding-top:60px;}
+    link(rel='stylesheet', href='/stylesheets/custom.css')
```

Comment:
True, but if you do work on this more you can add in new styling to the custom file.